sdk/src/types/block/mod.rs

```diff
@@ -42,12 +42,12 @@ pub mod slot;
 pub mod unlock;
 
 #[cfg(feature = "serde")]
-pub use self::core::dto::{BlockDto, BlockWrapperDto};
+pub use self::core::dto::{BlockDto, SignedBlockDto, UnsignedBlockDto};
 pub(crate) use self::r#macro::*;
 pub use self::{
     block_id::{BlockHash, BlockId},
     convert::ConvertTo,
-    core::{Block, BlockWrapper},
+    core::{Block, SignedBlock},
```

Comment:
Do we also want/need to expose UnsignedBlock? Since you're exposing UnsignedBlockDto